tests/unit/types/argument-set-test.js

```diff
@@ -49,9 +56,19 @@ test("it can be made from a query", function(assert) {
   assert.equal(set[1].value, 10);
 });
 
-test("filters out null arguments", function(assert) {
+test("does not filter out null arguments from a query", function(assert) {
   let set = ArgumentSet.fromQuery({ status: null, limit: 10 });
```

Comment:
Would be useful to have more falsy values in the test like empty string, number zero, etc...